.circleci/config.yml

```diff
@@ -119,7 +120,10 @@ workflows:
   version: 2
   build_and_e2eTest:
     jobs:
-      - build
+      - build:
+          filters:
+            tags:
+              only: /^v.*/
```

Comment:
Does it make any sense to use stricter pattern here? Something like `/^v[0-9]*\.[0-9]*\.[0-9]*$/`

Reply:
I have not a strong opinion on this, but I can add this. Actually the regex should also match `-<sha>` suffix:  `/^v[0-9]\.[0-9]\.[0-9].*` as we

Reply:
Yeah, I don't have strong opinions on this either. I was just wondering if it could be more strict, but when there's SHA postfix, I guess your initial version is just fine as well.

Reply:
Being more strict make sense as we only want to have tag following this pattern. I updated the filter.